src/pretalx/event/models/event.py

```diff
@@ -138,6 +138,8 @@ class orga_urls(Urls):
         create = '/orga/event/new'
         base = '/orga/event/{self.slug}'
         cfp = '{base}/cfp'
+        view_cfp = '{cfp}/text'
```

Comment:
These are accessible at `event.cfp.urls.text` and `event.cfp.urls.edit_text` at the moment.